manifests/params.pp

```diff
@@ -121,5 +121,13 @@
   $sites_available_group = $_module_parameters['root_group']
   $sites_available_mode  = '0644'
   $super_user            = true
+  if fact('nginx_version') {
+    $add_listen_directive = versioncmp(fact('nginx_version'), '1.15.0') ? {
```

Comment:
Isn't this equal to `$add_listen_directive = versioncmp(fact('nginx_version'), '1.15.0') < 0`?

Reply:
I has some issues in the past with versioncmp() when the first param is undef, that's why I wrapped it within the if statement.

Reply:
I meant the `? { ...}` part. The full oneliner would be `$add_listen_directive = !fact('nginx_version') or versioncmp(fact('nginx_version'), '1.15.0') < 0` but that's not really readable.